plenum/server/replica.py

```diff
@@ -204,7 +210,7 @@ def __init__(self, node: 'plenum.server.node.Node', instId: int,
         # viewNo and value a map of pre-prepare sequence number to commit
         self.stashed_out_of_order_commits = {}  # type: Dict[int,Dict[int,Commit]]
 
-        self.checkpoints = SortedDict(lambda k: k[0])
+        self.checkpoints = SortedDict(lambda k: k[1])
```

Comment:
Why don't we sort using 3PC comparison utility method? If the field used for sorting is not essential, then should we really use SortedDict here?